content/en/docs/migration/opentracing.md

```diff
@@ -1,73 +1,47 @@
 ---
 title: Migrating from OpenTracing
-weight: 20
+linkTitle: OpenTracing
 draft: true
 spelling: cSpell:ignore codebases
 ---
 
-The OpenTelemetry project aims to provide backwards compatibility with the
-[OpenTracing][] project, in order to ease migration to
-OpenTelemetry. The intention is to allow OpenTracing instrumentation to be
-recorded using the OpenTelemetry SDK, and to allow both the OpenTelemetry
-and the OpenTracing APIs to be used together in the same codebase.
+Backwards compatibility with [OpenTracing][] has been a priority for the
+OpenTelemetry project from the start. To ease migration, OpenTelemetry supports
+the use of both the OpenTelemetry _and_ OpenTracing APIs in the same codebase.
+This allows OpenTracing instrumentation to be recorded using OpenTelemetry SDKs.
 
-To accomplish this, each OpenTelemetry SDK provides an **OpenTracing Shim**,
-which can be installed in order to connect the SDK to the OpenTracing API.
-Please note that the OpenTracing Shim is not enabled by default.
+To accomplish this, each OpenTelemetry SDK provides an **[OpenTracing
+shim][OT_spec]**, which acts as a bridge between the OpenTracing API and the
+OpenTelemetry SDK. Note that the OpenTracing shim is disabled by default.
 
-## Language version support
```

Comment:
As I mentioned in the opening comment, this has been moved to **Limits on compatibility**, since IMHO that's where it makes the most sense.

Reply:
This table is how we are currently linking to the existing shims, and the docs for how to install them – something like that should be at the top so that users can find the actual implementations.

Reply:
Ok, moving this back to the top.